app/templates/index.hbs

```diff
@@ -36,18 +36,24 @@
     <li class="item list-unstyled">
       <EmberVersionGraphic @mascot="ember" />
       <ul class="links">
-        <li class="list-unstyled" data-test-ember-1-link>
+        <div class="row">
```

Comment:
I think that a `ul` must have `li` as direct children, without divs in between, based on the "Permitted content" section of the `ul` docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#technical_summary

One way to achieve this layout without the divs, while still keeping this one continuous list, is something like https://codepen.io/srikarg/pen/kxwQXV

There are lots of options for how to write this CSS. Another example is with flexbox: https://stackoverflow.com/questions/42613350/how-can-i-create-multi-columns-from-a-single-unordered-list

Reply:
Got it thanks! I'll push up a few commits to address this when I get a chance